bank/client/rest/tx.go

```diff
@@ -19,6 +19,7 @@ func RegisterRoutes(cliCtx context.CLIContext, r *mux.Router) {
 	r.HandleFunc("/bank/accounts/{address}/transfers", SendRequestHandlerFn(cliCtx)).Methods("POST")
 	r.HandleFunc("/bank/accounts/topup", TopupHandlerFn(cliCtx)).Methods("POST")
 	r.HandleFunc("/bank/balances/{address}", QueryBalancesRequestHandlerFn(cliCtx)).Methods("GET")
+	r.HandleFunc("/bank/accounts/withdrawtopup", WithdrawTopupHandlerFn(cliCtx)).Methods("POST")
```

Comment:
Can we do `/bank/accounts/topup/withdraw`?

Reply:
Let's not call the fee as `topup` anywhere, `topup` should be used to refer to the method via which a validator can add funds to his account to be used as fees. Otherwise we always refer the `MATIC` tokens on heimdall as `MATIC` and represent them as simple `balance`